chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/listing/pending/PendingCertifiedProductDTO.java

```diff
@@ -65,8 +73,10 @@
     private Long productClassificationId;
     private Boolean deleted;
     private Long lastModifiedUser;
-    private Set<String> errorMessages;
-    private Set<String> warningMessages;
+    @Builder.Default
```

Comment:
Does this work? I tried something like this with something, and got an NPE when trying to add to this array

Reply:
Same question. This was originally added in a ticket for OCD-3245 and is in what I merged into qa this morning during the reset.